.ci/lib.sh

```diff
@@ -59,3 +59,34 @@ function get_cc_versions(){
 	[ ! -f "$_versions_file" ] && { echo >&2 "ERROR: cannot find $_versions_file"; exit 1; }
 	source "$_versions_file"
 }
+
+
+function apply_depends_on() {
+	pushd "${GOPATH}/src/${kata_repo}"
+	label_lines=$(git log --format=%s%b -n 1 | grep "Depends-on:" || true)
+	if [ "${label_lines}" == "" ]; then
+		return 0
+	fi
+
+	nb_lines=$(echo "${label_lines}" | wc -l)
+
+	for i in $(seq 1 "${nb_lines}")
+	do
+		label_line=$(echo "${label_lines}" | sed "${i}q;d")
+		label_str=$(echo "${label_line}" | cut -d' ' -f2)
```

Comment:
This line assumes that the commit line starts with `Depends-on:` - any leading whitespace will break that. I'd either document this in the `README.md` or use `awk` instead of `cut` maybe.

I'd actually be tempted to remove `Depends-on:` from the string as soon as you have a match, then remove any leading/trailing whitespace and only then start trying to pick apart the repo+pr elements.

Reply:
Fixed using `awk`